build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.8.21"
+    application
+    id("org.jetbrains.kotlinx.kover") version "0.7.3"
+}
+
+group = "net.maizegenetics"
+version = "2.0.0"
+
+/*
+This build script is need to use the early access
+ */
+buildscript {
+    val kotlinVersion by extra("1.8.21")
+
+    repositories {
+        mavenCentral()
+        gradlePluginPortal()
+    }
+
+    dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
+        classpath(kotlin("serialization", version = kotlinVersion))
+        classpath("org.jetbrains.dokka:dokka-gradle-plugin:1.6.21")
+    }
+}
+
+repositories {
+    mavenCentral()
+    gradlePluginPortal()
+    maven("https://maven.imagej.net/content/groups/public/")
+}
+
+dependencies {
+    //testImplementation(kotlin("test"))
+    val kotlinVersion = rootProject.extra["kotlinVersion"]
+
+    implementation("org.biokotlin:biokotlin:0.08")
+
+    implementation("com.github.ajalt.clikt:clikt:4.2.0")
+
+    implementation("com.github.samtools:htsjdk:4.0.1")
+
+    implementation("org.jetbrains.kotlin:kotlin-test:1.9.10")
+    implementation("org.jetbrains.kotlin:kotlin-reflect:${kotlinVersion}")
+    implementation("org.jetbrains.kotlin:kotlin-script-runtime:${kotlinVersion}")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.5.2")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.2.2")
+    implementation("org.jetbrains.kotlinx:dataframe:0.8.0-rc-7")
```

Comment:
For the scope of this application - do we need this? Is it used in the original PHG source?

Reply:
^^^ the `dataframe` library, specifically

Reply:
kotlin-script allows Kotlin Scripts, so I would keep that. kotlinx-coroutines is for multithread stuff. I'm not sure about serialization.

I can get rid of dataframe? It can be added later if we need it

Reply:
The other kotlinx dependencies make sense - let's get rid of `dataframe` for now. Like you said, we can add it later it is really needed

Reply:
done